jupyterhub/tests/test_roles.py

```diff
@@ -571,7 +571,8 @@ async def test_load_roles_groups(tmpdir, request):
     hub.init_db()
     db = hub.db
     await hub.init_groups()
-    await hub.init_roles()
+    await hub.init_role_creation()
```

Comment:
Let's try to preserve the order - creation should happen before groups

Reply:
Good idea, thanks